.circleci/config.yml

```
@@ -658,6 +712,13 @@ workflows:
           context: Slack Orb
       - spm-core-integration-test-job:
           name: "Xcode 13; iOS 15.0; SPM Core test"
+      - generate-docs-job:
+          name: Generate docs and push it for release tag
```

This is a bit verbose compared to other job names.

```suggestion
          name: Generate/Publish Documentation
```